src/win32/custom_action/custom_action.cc

```diff
@@ -328,6 +333,18 @@ UINT __stdcall EnableTipProfile(MSIHANDLE msi_handle) {
   return ERROR_SUCCESS;
 }
 
+UINT __stdcall FixupConfigFilePermission(MSIHANDLE msi_handle) {
```

Comment:
Does this always return `ERROR_SUCCESS`? How about mentioning it to clarify the behavior?

Reply:
Done.